src/frontend/i18n/fr_FR.json

```diff
@@ -1,4 +1,28 @@
 {
+  "JitsiControls.AudioControl.disableTitle": {
+    "description": "Title for jitsi mute/unmute button when audio is off",
+    "message": "Remettre le son"
+  },
+  "JitsiControls.AudioControl.enableTitle": {
+    "description": "Title for jitsi mute/unmute button when audio is on",
+    "message": "Couper le son"
```

Comment:
J'adore...

Reply:
we should translate this to:
- Je coupe le son
- Je remets le son

Reply:
https://www.youtube.com/watch?v=nLN8pvR8hDg